### ISSUE TITLE
não funciona com mais de uma tela

### ISSUE THREAD

**Issue by exebixel** [fredcox/gonhang#4]
Quando se usa o gonhang em mais de uma tela não é possivel escolher em qual tela o gonhang vai ficar.

E ele sempre abre no meio entre as 2 telas 

![image](https://user-images.githubusercontent.com/58310340/103175934-afdad280-484c-11eb-9428-14ed9c34e153.png)

Estou usando como podem ver na foto o arch com kde

Seria bom se tivesse uma opção para escolher em qual tela o gonha apareceria

**Comment by fredcox** [comment]
Sim, vc tem razão, na próxima atualização colocarei essa feature. Obrigado pelo report.

**Comment by exebixel** [comment]
obrigado!
